src/lib/consent-components/services/markdown/markdown.service.ts

```diff
@@ -0,0 +1,54 @@
+import { Inject, Injectable } from "@angular/core";
+import { ConsentModuleConfigService } from "../consent-module-config.service";
+import { MarkdownConfig } from "./markdown-config";
+import { MarkdownType } from "./markdown-type.enum";
+
+@Injectable({
+  providedIn: "root",
+})
+export class MarkdownService {
+  private markdownBaseUrl = "";
+  private countrySuffix = "";
+  private markdownNames: { [key in keyof typeof MarkdownType]: string } = {
+    SmartlookConsent: "smartlook-info.md",
```

Comment:
Should urls to .md files be specific to adia? This library is for Adia only use, or is Open Source so anyone could use it? Maybe you could just provide urls to this lib from our web apps?

Reply:
I assumed that's open source so I removed URL from here. It could be done simpler with URLs but i have no more time now for refactoring.